src/transformers/models/auto/modeling_auto.py

```diff
@@ -719,6 +719,26 @@
     ]
 )
 
+MODEL_FOR_IMAGE_TEXT_TO_TEXT_MAPPING_NAMES = OrderedDict(
+    [
+        ("blip", "BlipForConditionalGeneration"),
+        ("blip-2", "Blip2ForConditionalGeneration"),
+        ("fuyu", "FuyuForCausalLM"),
```

Comment:
Chameleon can also do Image-text to text
```suggestion
        ("blip-2", "Blip2ForConditionalGeneration"),
        ("chameleon", "ChameleonForConditionalGeneration"),
        ("fuyu", "FuyuForCausalLM"),
```

Reply:
+1 - though it can also do image-text to image, do we want it in this mapping still?